code/frameworks/nextjs/src/preset.ts

```diff
@@ -73,10 +77,63 @@ export const previewAnnotations: StorybookConfig['previewAnnotations'] = (entry
   require.resolve('@storybook/nextjs/preview.js'),
 ];
 
-// export const webpackInstance = async () => {
-//   webpack.init();
-//   return webpack;
-// };
+// Not even sb init - automigrate - running dev
+// You're using a version of Nextjs prior to v10, which is unsupported by this framework.
+
+export const babel = async (baseConfig: TransformOptions): Promise<TransformOptions> => {
+  const configPartial = loadPartialConfig({
```

Comment:
does this need a try/catch or is loadPartialConfig going to return `null` if there's no babel file?

Reply:
Same as above. I have just introduced this peace of code, which was removed in the first branch. There are already next.js projects out there without any babel config. So we support this case and this call doesn't fail in that case.